src/Tasks/Microsoft.NET.Build.Tasks/ProcessFrameworkReferences.cs

```diff
@@ -426,16 +428,11 @@ protected override void ExecuteCore()
                 }
             }
 
-            // Keep this in sync with _RequiresILLinkPack in Microsoft.NET.Publish.targets.
-            bool requiresILLinkPack = PublishAot
-                || IsAotCompatible || EnableAotAnalyzer
-                || PublishTrimmed
-                || IsTrimmable || EnableTrimAnalyzer
-                || EnableSingleFileAnalyzer;
-            if (requiresILLinkPack)
+            if (RequiresILLinkPack)
```

Comment:
Would it make sense to generate the errors outside of the `ProcessFrameworkReferences` task?  Then I tihnk we could remove all of these additional task parameters that were added.

Reply:
Thanks for looking at it.  If it seems like there's not an obvious way to simplify it well, then we can leave it as it is.